specification/subscription/resource-manager/Microsoft.Subscription/preview/2021-01-01-preview/examples/changeTenantPolicy.json

```diff
@@ -0,0 +1,31 @@
+{
+  "parameters": {
+    "api-version": "2021-01-01-preview",
+    "body": {
+      "blockSubscriptionsLeavingTenant": true,
+      "blockSubscriptionsIntoTenant": true,
+      "exemptedPrincipals": [
+        "e879cf0f-2b4d-5431-109a-f72fc9868693",
+        "9792da87-c97b-410d-a97d-27021ba09ce6"
+      ]
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "providers/Microsoft.Subscription/policies/default",
+        "name": "default",
+        "type": "providers/Microsoft.Subscription/policies/default",
```

Comment:
type should be 'providers/Microsoft.Subscription/policies'

Reply:
Updated